app/services/stellar_base/deposit_requests/find_config.rb

```diff
@@ -0,0 +1,23 @@
+module StellarBase
+  module DepositRequests
+    class FindConfig
+
+      extend LightService::Action
+
+      expects :network
+      promises :deposit_config
+
+      executed do |c|
+        depositable_assets = StellarBase.configuration.depositable_assets.dup
```

Comment:
Why `dup`?

Reply:
Oops, no need for that. Thanks for the catch